src/Covenant/DeBruijn.hs

```diff
@@ -81,3 +82,6 @@ asInt (DeBruijn i) = fromIntegral i
 
 reduce :: DeBruijn -> Maybe DeBruijn
 reduce (DeBruijn x) = DeBruijn (x - 1) <$ guard (x > 0)
+
+unsafeDeBruijn :: Int -> DeBruijn
+unsafeDeBruijn = DeBruijn . fromIntegral
```

Comment:
This should probably live with the non-helpers (above the `-- Helper`) comment, be `@since`d and documented. I would prefer instead if `asInt` became a prism, since then it could be `fromJust`ed on location.

Reply:
Did the prism thing & made all the changes everywhere else. You're right, it makes more sense as a prism.